# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/loader/ModuleLoaderBase.h
# Commit: 52a87b60e974
# Full Hash: 52a87b60e9740dbde9a4d580c200f7c07c550a6f
# Author: Yoshi Cheng-Hao Huang <allstars.chh@gmail.com>
# Date: 2025-09-26 06:07 +0000
# Regressor Bug: 1968890
# File Overlap Count: 2
# Overlapping Files: js/loader/ModuleLoaderBase.h, js/loader/ModuleLoaderBase.cpp
# Description:
#   Bug 1968890 - Part 4: Handle Dynamic import in LoadRequestedModulesRejected. r=jonco
# ==============================================================================

diff -r a20ad923b1f8 -r 52a87b60e974 js/loader/ModuleLoaderBase.h
--- a/js/loader/ModuleLoaderBase.h	Fri Sep 26 06:07:43 2025 +0000
+++ b/js/loader/ModuleLoaderBase.h	Fri Sep 26 06:07:44 2025 +0000
@@ -549,7 +549,8 @@
                                              Handle<Value> aHostDefined,
                                              Handle<Value> aError);
   static bool OnLoadRequestedModulesResolved(ModuleLoadRequest* aRequest);
-  static bool OnLoadRequestedModulesRejected(ModuleLoadRequest* aRequest,
+  static bool OnLoadRequestedModulesRejected(JSContext* aCx,
+                                             ModuleLoadRequest* aRequest,
                                              Handle<Value> aError);
 
   /**
